website/src/pages/auth/confirm-signup.page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import {
+  unstable_Form as Form,
+  unstable_useFormState as useFormState,
+} from "reakit"
+import { useUser } from "src/auth"
+import {
+  FormFields,
+  FormSubmitButton,
+  UserAuthPageTemplate,
+} from "./user-auth-layout"
+import { centeredForm } from "./user-auth.css"
+
+const VerificationPage = () => {
```

Comment:
How do I ever get back to this page if I close the tab I started this workflow in? Is it included in the email?

Reply:
Also your file and main component are fighting over what this page is called. Are we confirming the sign up or are we verifying the account?